src/components/Header.vue

```diff
@@ -67,6 +67,12 @@
         <LockClosed class="w-4 h-4 mr-2" />
         Commit
       </button>
+      <button
+        class="flex items-center font-light p-2 bg-blue-100 rounded-t px-2 ml-4"
+        @click="logout"
+      >
+        Logout
```

Comment:
I'd add an icon here, like Heroicon's `heroicons/outline/logout.svg`. If you don't want to add an icon, drop `flex items-center` they don't do anything.